release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -88,6 +88,11 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-9-builds"]
 === Builds
 
+[id="ocp-4-9-builds-image-changes"]
+==== Use ImageChange in the BuildConfig status to trigger builds
+
+As a developer configuring GitOps deployments, you can add `ImageChange` triggers to the `BuildConfig` status. This type of trigger initiates a build when the base image it references is updated.
```

Comment:
End users don't touch `status` in k8s objects - these should only be updated by controllers.

This feature means that developers can add BuildConfigs with ImageChange triggers to their OpenShift GitOps workflows. Image change triggers will not cause OpenShift GitOps to report itself as out of sync.